src/main/java/frc/robot/controllers/CustomController2020.java

```diff
@@ -11,9 +11,9 @@
         SHOOTER_AUTO(7),
         SHOOTER_FLYWHEEL(8),
         SHOOTER_SHOOT(9),
-        CLIMB_LOCK(10),
-        CLIMB_UP(11),
-        CLIMB_DOWN(12);
+        CLIMBER_LOCK(10),
+        CLIMBER_UP(11),
+        CLIMBER_DOWN(12);
```

Comment:
Would it be helpful to name these `CLIMBER_LOCK_SWITCH`, `CLIMBER_UP_BUTTON`, `CLIMBER_DOWN_BUTTON`?